config_only_jre.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023s the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ubi8javabuildpack
+
+import (
+	"github.com/buildpacks/libcnb/v2"
+	"github.com/paketo-buildpacks/libjvm/v2"
+	"github.com/paketo-buildpacks/libpak/v2"
+	"github.com/paketo-buildpacks/libpak/v2/log"
+)
+
+// consts for the config only JRE
+const isBuild = false
+const isLaunch = true
+const isCache = false
+const name = "ajre" //alphabetically must come before 'helper' and 'java-security-properties'.
```

Comment:
I'm curious about this. My understanding is that the name is just for display purposes.

Reply:
Going from memory here.. the issue is that each of these creates a Layer directory, and according to the Buildpack spec, if a single buildpack creates multiple layers, they are processed alphabetically .. originally I had this as just `jre` but that failed.. so I made it ajre and it resolved it..